node/sc/bridge_accounts.go

```diff
@@ -106,8 +103,19 @@ func (ba *BridgeAccounts) SetChildOperatorFeePayer(feePayer common.Address) erro
 	return nil
 }
 
+//SetBridgeGasLimit changes value of DefaultBridgeTxGasLimit.
```

Comment:
The name `DefaultBridgeTxGasLimit` seems for the variable in the test file. Could you check it out again?

Reply:
Yes, that `DefaultBridgeTxGasLimit` variable is only for test purpose. It's nothing to do with klaytn protocol code.

Reply:
The comment says that sets the fee to `DefaultBridgeTxGasLimit`, but the code does not actually. I meant to request you to review the comment again.

Reply:
I'll reflect your comment. Thank you.